src/shared/components/pagination/pagination.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+function coerceToBoolean(input: string | boolean): boolean {
+    return !!input && input !== 'false';
+}
+
+@Component({
+  selector: 'ccd-pagination',
+  templateUrl: './pagination.component.html',
+  styleUrls: ['./pagination.component.scss']
+})
+export class PaginationComponent {
+
+    @Input()
```

Comment:
4 space indenting? It's 2 in lines 8 to 10 and should be 2 throughout for consistency.